src/Stepper/Stepper.tsx

```diff
@@ -23,55 +20,38 @@ const propTypes = {
   })).isRequired,
   currentStepID: PropTypes.string.isRequired,
   onStepClick: PropTypes.func,
-  sticky: PropTypes.bool,
+  style: PropTypes.object,
```

Comment:
this prop was only needed to support the sticky container, so can drop it now - we want to encourage styling to happen via LESS at this point